### PR TITLE
New Bottom Navigation Design

### PR DESCRIPTION
**Pull request by harshvitra**
This adds a New Bottom Navigation Design.

Also the menu in this is not being fetched from the database. It is rendered directly from Front end.

### CODE REVIEW

File: app/containers/Menu/NewBottomNavigator.tsx [hunk 1]
```diff
@@ -13,18 +13,28 @@ const height = 64;
 const tabs = [
     {
         name: "account-outline",
+        title: "Me",
```
Comment (sagararyal) [inline app/containers/Menu/NewBottomNavigator.tsx]: please reuse the labels

File: app/containers/Menu/NewBottomNavigator.tsx [hunk 2]
```diff
@@ -13,18 +13,28 @@ const height = 64;
...
+        route: 'app_userHome'
     },
     {
         name: "gift-outline",
+        title: "Gift",
```
Comment (sagararyal) [inline app/containers/Menu/NewBottomNavigator.tsx]: same here

File: package.json [hunk 3]
```diff
@@ -39,6 +39,9 @@
     "cookie-cutter": "^0.2.0",
     "core-js": "^2.6.11",
     "css-loader": "^0.28.9",
+    "d3-interpolate-path": "^2.1.1",
+    "d3-path": "^1.0.9",
```
Comment (norbertschuler) [inline package.json]: Can find a usage of these two modules. Don't import unused modules increasing our bundle size.

**Comment by norbertschuler** [comment]
- [ ] While navigating (in the emulator it's slower then on a real device) the icons temporarily disappear:
[new_bottom_navi.mp4.zip](https://github.com/Plant-for-the-Planet-org/treecounter-app/files/4127500/new_bottom_navi.mp4.zip)

**Comment by norbertschuler** [comment]
- [ ] Initial screen still show text below "World" icon although it's selected and the "Me" icon is missing:

Screenshot_1580296976

**Pull request merged**